main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java

```diff
@@ -264,6 +274,11 @@ public void onCreate(final Bundle savedInstanceState) {
 
         viewModel.mapCenter.observe(this, center -> refreshListChooser());
```

Comment:
If we add an observer to the whole viewport, do we still need to observe map center?

Reply:
That's a very good point, and I don't think we need both. Also, the way that mapCenter is set/updated is much better than my approach. Thanks for the review, I will rework this part of the PR

Reply:
ok, fixed